cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -150,37 +153,11 @@ def test_courses_list_with_ccx_courses(self):
 
         # Verify that CCX courses are filtered out while iterating over all courses
         mocked_ccx_course = Mock(id=ccx_course_key)
-        with patch('xmodule.modulestore.mixed.MixedModuleStore.get_courses', return_value=[mocked_ccx_course]):
-            courses_iter, __ = _accessible_courses_iter(self.request)
+        with patch('xmodule.modulestore.mixed.MixedModuleStore.get_course_summaries', return_value=[mocked_ccx_course]):
+            courses_iter, __ = _accessible_courses_iter_for_tests(self.request)
             self.assertEqual(len(list(courses_iter)), 0)
 
     @ddt.data(
-        (ModuleStoreEnum.Type.split, 'xmodule.modulestore.split_mongo.split_mongo_kvs.SplitMongoKVS'),
-        (ModuleStoreEnum.Type.mongo, 'xmodule.modulestore.mongo.base.MongoKeyValueStore')
-    )
-    @ddt.unpack
-    def test_errored_course_global_staff(self, store, path_to_patch):
```

Comment:
Why this test is removed?

Reply:
This is testing course fetching by `get_course(course_key)` and related `ErrorDescriptor` return. We are no longer using this method to fetch course for listing so we don't need this test.